spiel/st/m0t1fs_spiel_sns_repair.sh

```diff
@@ -35,8 +35,8 @@ spiel_sns_repair_and_rebalance_test()
 
 	echo "Starting SNS repair testing ..."
 	for ((i=0; i < ${#files[*]}; i++)) ; do
-		touch_file $MOTR_M0T1FS_MOUNT_DIR/${files[$i]} ${unit_size[$i]}
-		_dd ${files[$i]} $((${unit_size[$i]} * 1024)) ${file_size[$i]}
+		touch_file "$MOTR_M0T1FS_MOUNT_DIR"/${files[$i]} ${unit_size[$i]}
+		_dd "${files["$i"]}" $(("${unit_size["$i"]}" * 1024)) "${file_size["$i"]}"
```

Comment:
This will not work. You are multiplying sting and integer. Did you test these changes?

Reply:
incorporated